Dockerfile

```diff
@@ -36,7 +42,6 @@ RUN userdel -r yoctouser && \
         /usr/bin/restrict_groupadd.sh \
         /usr/bin/restrict_useradd.sh && \
     echo "#include /etc/sudoers.usersetup" >> /etc/sudoers
-
```

Comment:
I don't know why I didn't mention this before, but could you add the blank line back in. I think you accidentally deleted it.

Reply:
Sure, no problem.